scapy/contrib/pnio.py

```diff
@@ -0,0 +1,78 @@
+# encoding: utf-8
+
+# This file is part of Scapy
+# Scapy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# any later version.
+#
+# Scapy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Scapy. If not, see <http://www.gnu.org/licenses/>.
+
+# Copyright (C) 2016 Gauthier Sebaux
+
+"""
+A simple and non exhaustive Profinet IO layer for scapy
+"""
+
+# Scapy imports
+# pylint: disable=no-name-in-module
+from scapy.all import Packet, bind_layers, Ether, UDP
+# pylint: enable=no-name-in-module
+from scapy.fields import XShortEnumField
+
+# Some constants
+PNIO_FRAME_IDS = {
+    0x0020:"PTCP-RTSyncPDU-followup",
```

Comment:
You should pick either ' our " to declare strings and use the same notation everywhere.